packages/fast-components/src/checkbox/checkbox.template.ts

```diff
@@ -0,0 +1,54 @@
+import { html, when } from "@microsoft/fast-element";
+import { Checkbox } from "./checkbox";
+import { bool } from "../utilities";
+
+export const CheckboxTemplate = html<Checkbox>`
+    <div
+        part="checkbox"
+        role="checkbox"
+        class="checkbox"
+        $aria-checked="${x => bool(x.checked)}"

```

Comment:
I think we should put these aria attributes on the host element. We can circle back and do that after my PR for host instructions lands.

Reply:
I'm a little surprised narrator read the right thing when I tried this because the label isn't associated with the host element - but it seems to read any child content so I'm good with this. I think the full scope of this change will be not deferring focus on this control, making the host the focus target and boosting relevant attributes to the host - does that sound right. We also probably don't need to add "checked" or "disabled" classes, but we still could if that is a good utility.

Reply:
hoisted to the parent. Left class-names in place so that styling overrides capabilities are separated from component internal implementation.